.github/workflows/docker-image.yml

```diff
@@ -17,6 +17,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      # https://github.com/marketplace/actions/docker-buildx
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
```

Comment:
what is the licence for this? can you provide any information about security audits it has passed?

Reply:
License is MIT: https://github.com/crazy-max/ghaction-docker-buildx/blob/master/LICENSE

I couldn't find anything regarding a security audit tbh

Reply:
MIT is fine - thank you